src/cookieconsent.js

```diff
@@ -132,6 +134,24 @@
       // The code "typeof obj === 'object' && obj !== null" allows Array objects
       return typeof obj === 'object' && obj !== null && obj.constructor == Object;
     },
+
+    loadStylesheet: function (stylesheet, complete) {
+      var link = document.createElement("link");
```

Comment:
This is optional. It's not used by default, but if you pass a `stylesheet` property, it will dynamically load the stylesheet. This is useful if you want to defer the loading of the script, but don't want to add the CSS handling code